pkg/controller/reconcile.go

```diff
@@ -55,14 +55,34 @@ func (r *AnsibleOperatorReconciler) Reconcile(request reconcile.Request) (reconc
 		return reconcile.Result{}, nil
 	}
 
-	s := u.Object["spec"]
-	_, ok := s.(map[string]interface{})
+	spec := u.Object["spec"]
+	_, ok := spec.(map[string]interface{})
 	if !ok {
 		logrus.Warnf("spec was not found")
 		u.Object["spec"] = map[string]interface{}{}
 		r.Client.Update(context.TODO(), u)
 		return reconcile.Result{Requeue: true}, nil
 	}
+	status := u.Object["status"]
+	_, ok = status.(map[string]interface{})
+	if !ok {
+		logrus.Warnf("status was not found")
```

Comment:
We may want to make these debug in the future. 

nothing is wrong it is expected status is not found at first, so this might be the wrong level (sorry for the bad example above)